Arduino/Finger_Test/Finger_Test.ino

```diff
@@ -18,13 +18,16 @@ void setup()
 	pinMode(Finger_WAKE_Pin , INPUT);
 	Finger_SoftwareSerial_Init();	
 	Finger_Wait_Until_OK();
+
+  //Enter Sleep Mode
+  digitalWrite(Finger_RST_Pin , LOW);
+  Finger_SleepFlag = 1;
```

Comment:
These two lines are the same exact thing that CMD5 does to entere sleep mode. Yes it is redundant with the code just above it. This simply makes it clearer.